airflow/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -125,6 +128,34 @@ class DbtCloudJobRunException(AirflowException):
     """An exception that indicates a job run failed to complete."""
 
 
+T = TypeVar("T", bound=Any)
+
+
+def provide_account_id(func: T) -> T:
+    """
+    Decorator which provides a fallback value for ``account_id``. If the ``account_id`` is None or not passed
+    to the decorated function, the value will be taken from the configured dbt Cloud Airflow Connection.
+    """
+    function_signature = signature(func)
```

Comment:
The problem with this wrapper (as well as other same decorators which already exists in community providers) that every time we access to method which decorated by this function we need to lookup for connection in Secrets Backend (if configured), Environment Variables and Airflow Database.

Is any way to make it cashable? Something like that: https://github.com/apache/airflow/pull/28716

Reply:
will check if it can be made cacheable